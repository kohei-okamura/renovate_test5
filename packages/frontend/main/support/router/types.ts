/*
 * Copyright Â© 2020 EUSTYLE LABORATORY - ALL RIGHTS RESERVED.
 * UNAUTHORIZED COPYING OF THIS FILE, VIA ANY MEDIUM IS STRICTLY PROHIBITED PROPRIETARY AND CONFIDENTIAL.
 */
import { Route } from 'vue-router'

export type RouteParams = Route['params']

export type RouteQuery = Route['query']
