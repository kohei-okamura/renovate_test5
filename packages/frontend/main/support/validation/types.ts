/*
 * Copyright Â© 2019 EUSTYLE LABORATORY - ALL RIGHTS RESERVED.
 * UNAUTHORIZED COPYING OF THIS FILE, VIA ANY MEDIUM IS STRICTLY PROHIBITED PROPRIETARY AND CONFIDENTIAL.
 */
import { ValidationObserver, ValidationProvider } from 'vee-validate'
import { DateLike } from '~/models/date'
import { CustomRuleParams } from '~/support/validation/rules/custom'

type BetweenRule = {
  min: number
  max: number
}

type BooleanRule = boolean

type CustomRule = CustomRuleParams

type MinDateRule =
  (DateLike | undefined) |
  [DateLike | undefined] |
  [DateLike | undefined, string] |
  {
    minValue: DateLike | undefined
    attribute?: string
  }

type LengthRule = number | {
  len: number
  max?: number
}

type NumericRule = number | string

type NonItemsZeroRule = {
  itemName: string
}

export type Rule = {
  asciiAlphaNum?: BooleanRule
  alphaNum?: BooleanRule
  between?: BetweenRule
  custom?: CustomRule
  digits?: NumericRule
  email?: BooleanRule
  fax?: BooleanRule
  hiragana?: BooleanRule
  length?: LengthRule
  katakana?: BooleanRule
  max?: NumericRule
  min?: NumericRule
  minDate?: MinDateRule
  minValue?: NumericRule
  nonItemsZero?: NonItemsZeroRule
  integer?: BooleanRule
  numeric?: BooleanRule
  postcode?: BooleanRule
  required?: BooleanRule
  tel?: BooleanRule
  validTimeDuration?: BooleanRule
  zenginDataRecordChar?: BooleanRule
}

export type Rules = {
  [key: string]: Rule | Rule[] | Rules | Rules[] | undefined
}

export type ValidationObserverInstance = InstanceType<typeof ValidationObserver>

export type ValidationProviderInstance = InstanceType<typeof ValidationProvider>
