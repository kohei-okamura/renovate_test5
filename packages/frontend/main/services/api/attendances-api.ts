/*
 * Copyright © 2020 EUSTYLE LABORATORY - ALL RIGHTS RESERVED.
 * UNAUTHORIZED COPYING OF THIS FILE, VIA ANY MEDIUM IS STRICTLY PROHIBITED PROPRIETARY AND CONFIDENTIAL.
 */
import { ServiceOption } from '@zinger/enums/lib/service-option'
import { Task } from '@zinger/enums/lib/task'
import { AxiosInstance } from 'axios'
import { Assignee } from '~/models/assignee'
import { Attendance, AttendanceId } from '~/models/attendance'
import { ContractId } from '~/models/contract'
import { Duration } from '~/models/duration'
import { Job } from '~/models/job'
import { OfficeId } from '~/models/office'
import { Schedule } from '~/models/schedule'
import { StaffId } from '~/models/staff'
import { UserId } from '~/models/user'
import { api, Api } from '~/services/api/core'

/**
 * 勤務実績 API.
 */
export namespace AttendancesApi {
  export type Form = {
    task: Task
    serviceCode: string
    officeId: OfficeId | undefined
    userId: UserId | undefined
    contractId: ContractId | undefined
    assignerId: StaffId
    assignees: Assignee[]
    headcount: number
    schedule: Schedule
    durations: Duration[]
    options: ServiceOption[]
    note: string
  }

  export type GetParams = Api.GetParams
  export type GetResponse = {
    attendance: Attendance
  }

  export type GetIndexParams = Api.GetIndexParams & {
    start?: string
    end?: string
    officeId?: number | ''
    userId?: number | ''
    assigneeId?: number | ''
    assignerId?: number | ''
    isConfirmed?: boolean | ''
    task?: Task | ''
  }
  export type GetIndexResponse = Api.GetIndexResponse<Attendance>

  export type UpdateResponse = GetResponse

  export type BatchResponse = {
    job: Job
  }

  export type BatchCancel = {
    batchCancel (params: { ids: AttendanceId[], reason: string }): Promise<BatchResponse>
  }

  export type Cancel = {
    cancel (params: { id: AttendanceId, reason: string }): Promise<void>
  }

  export type Confirm = {
    confirm (params: { ids: AttendanceId[] }): Promise<BatchResponse>
  }

  export type Definition =
    BatchCancel &
    Cancel &
    Confirm &
    Api.Create<Form> &
    Api.Get<GetResponse> &
    Api.GetIndex<GetIndexParams, GetIndexResponse> &
    Api.Update<Form, Api.UpdateParams<Form>, UpdateResponse>

  const endpoint = (...segments: (string | AttendanceId)[]) => api.endpoint('attendances', ...segments)

  export const create = (axios: AxiosInstance): Definition => ({
    async batchCancel (params) {
      return await api.extract(axios.post(endpoint('cancel'), { params }))
    },
    async cancel ({ id, reason }) {
      await axios.post(endpoint(id, 'cancel'), { reason })
    },
    async confirm (params) {
      return await api.extract(axios.post(endpoint('confirmation'), params))
    },
    async create ({ form }) {
      await axios.post(endpoint(), form)
    },
    async get ({ id }) {
      return await api.extract(axios.get(endpoint(id)))
    },
    async getIndex (params) {
      return await api.extract(axios.get(endpoint(), { params }))
    },
    async update ({ form, id }) {
      return await api.extract(axios.put(endpoint(id), form))
    }
  })
}
