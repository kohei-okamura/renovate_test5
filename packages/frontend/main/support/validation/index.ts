/* eslint-disable camelcase */
/*
 * Copyright © 2019 EUSTYLE LABORATORY - ALL RIGHTS RESERVED.
 * UNAUTHORIZED COPYING OF THIS FILE, VIA ANY MEDIUM IS STRICTLY PROHIBITED PROPRIETARY AND CONFIDENTIAL.
 */
import { configure, extend, localize, ValidationObserver, ValidationProvider } from 'vee-validate'
import {
  alpha_num as alphaNum,
  between,
  digits,
  email,
  integer,
  length,
  max,
  max_value as maxValue,
  min,
  min_value as minValue,
  numeric,
  required
} from 'vee-validate/dist/rules'
import { VueConstructor } from 'vue'
import { asciiAlphaNum } from '~/support/validation/rules/ascii-alpha-num'
import { custom } from '~/support/validation/rules/custom'
import { fax } from '~/support/validation/rules/fax'
import { hiragana } from '~/support/validation/rules/hiragana'
import { katakana } from '~/support/validation/rules/katakana'
import { minDate } from '~/support/validation/rules/min-date'
import { nonItemsZero } from '~/support/validation/rules/non-items-zero'
import { postcode } from '~/support/validation/rules/postcode'
import { tel } from '~/support/validation/rules/tel'
import { validTimeDuration } from '~/support/validation/rules/valid-time-durations'
import { zenginDataRecordChar } from '~/support/validation/rules/zengin-data-record-char'

function setupComponents (vue: VueConstructor): void {
  vue.component('ValidationObserver', ValidationObserver)
  vue.component('ValidationProvider', ValidationProvider)
}

function setupRules (): void {
  extend('alphaNum', alphaNum)
  extend('asciiAlphaNum', asciiAlphaNum)
  extend('between', between)
  extend('custom', custom)
  extend('digits', digits)
  extend('email', email)
  extend('fax', fax)
  extend('hiragana', hiragana)
  extend('integer', integer)
  extend('katakana', katakana)
  extend('length', length)
  extend('max', max)
  extend('maxValue', maxValue)
  extend('min', min)
  extend('minDate', minDate)
  extend('minValue', minValue)
  extend('nonItemsZero', nonItemsZero)
  extend('numeric', numeric)
  extend('postcode', postcode)
  extend('required', required)
  extend('tel', tel)
  extend('validTimeDuration', validTimeDuration)
  extend('zenginDataRecordChar', zenginDataRecordChar)
}

function setupMessages (): void {
  localize('ja', {
    messages: {
      alphaNum: '半角英数字で入力してください',
      between: (_, params = {}) => `${params.min}以上、${params.max}以下の半角数字で入力してください。`,
      digits: (_, params = {}) => `${params.length}桁の半角数字で入力してください。`,
      email: '有効なメールアドレスを入力してください。',
      integer: '半角数字のみで入力してください。',
      length: (_, params = {}) => `${params.length}文字で入力してください。`,
      max: (_, params = {}) => `${params.length}文字以内で入力してください。`,
      maxValue: (_, params = {}) => `${params.max}以下を入力してください。`,
      min: (_, params = {}) => `${params.length}文字以上で入力してください。`,
      minValue: (_, params = {}) => `${params.min}以上を入力してください。`,
      numeric: '半角数字のみで入力してください。',
      required: '入力してください。'
    }
  })
}

/**
 * VeeValidate 関連のセットアップ処理.
 */
export function setupVeeValidate (vue: VueConstructor): void {
  configure({
    mode: 'eager'
  })
  setupComponents(vue)
  setupRules()
  setupMessages()
}
