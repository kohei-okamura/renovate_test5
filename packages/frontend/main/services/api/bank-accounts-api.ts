/*
 * Copyright © 2020 EUSTYLE LABORATORY - ALL RIGHTS RESERVED.
 * UNAUTHORIZED COPYING OF THIS FILE, VIA ANY MEDIUM IS STRICTLY PROHIBITED PROPRIETARY AND CONFIDENTIAL.
 */
import { BankAccountType } from '@zinger/enums/lib/bank-account-type'
import { AxiosInstance } from 'axios'
import { BankAccount } from '~/models/bank-account'
import { api, Api } from '~/services/api/core'

/**
 * 銀行口座 API.
 */
export namespace BankAccountsApi {
  export type Form = {
    userId?: number
    staffId?: number
    bankName?: string
    bankCode?: string
    bankBranchName?: string
    bankBranchCode?: string
    bankAccountType?: BankAccountType
    bankAccountNumber?: string
    bankAccountHolder?: string
  }

  export type UpdateResponse = {
    bankAccount: BankAccount
  }

  export type Definition = Api.Update<Form, Api.Form<Form>, UpdateResponse>

  export const create = (axios: AxiosInstance): Definition => ({
    async update ({ form }) {
      if (form.userId && form.staffId) {
        throw new Error('Parameter has multiple id')
      } else if (form.userId) {
        return await api.extract(axios.put(api.endpoint('users', form.userId, 'bank-account'), form))
      } else if (form.staffId) {
        return await api.extract(axios.put(api.endpoint('staffs', form.staffId, 'bank-account'), form))
      } else {
        throw new Error('Parameter has no id')
      }
    }
  })
}
