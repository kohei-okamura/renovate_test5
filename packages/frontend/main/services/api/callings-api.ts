/*
 * Copyright © 2020 EUSTYLE LABORATORY - ALL RIGHTS RESERVED.
 * UNAUTHORIZED COPYING OF THIS FILE, VIA ANY MEDIUM IS STRICTLY PROHIBITED PROPRIETARY AND CONFIDENTIAL.
 */
import { AxiosInstance } from 'axios'
import { Shift } from '~/models/shift'
import { api, Api } from '~/services/api/core'

/**
 * スタッフ：出勤確認 API.
 */
export namespace CallingsApi {
  export type AcknowledgeParams = Api.Token
  export type Acknowledge = {
    acknowledge (token: string): Promise<void>
  }

  export type GetIndex = {
    getIndex (token: string): Promise<Api.GetIndexResponse<Shift>>
  }

  export type Definition = Acknowledge & GetIndex

  const endpoint = (...segments: string[]) => api.endpoint('callings', ...segments)

  export const create = (axios: AxiosInstance): Definition => ({
    async acknowledge (token) {
      await axios.post(endpoint(token, 'acknowledges'))
    },
    async getIndex (token) {
      return await api.extract(axios.get(endpoint(token, 'shifts')))
    }
  })
}
