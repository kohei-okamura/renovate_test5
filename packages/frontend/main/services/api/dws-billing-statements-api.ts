/*
 * Copyright © 2020 EUSTYLE LABORATORY - ALL RIGHTS RESERVED.
 * UNAUTHORIZED COPYING OF THIS FILE, VIA ANY MEDIUM IS STRICTLY PROHIBITED PROPRIETARY AND CONFIDENTIAL.
 */
import { CopayCoordinationResult } from '@zinger/enums/lib/copay-coordination-result'
import { DwsBillingStatementCopayCoordinationStatus } from '@zinger/enums/lib/dws-billing-statement-copay-coordination-status'
import { DwsBillingStatus } from '@zinger/enums/lib/dws-billing-status'
import { AxiosInstance } from 'axios'
import { DwsBilling, DwsBillingId } from '~/models/dws-billing'
import { DwsBillingBundle, DwsBillingBundleId } from '~/models/dws-billing-bundle'
import { DwsBillingStatement, DwsBillingStatementId } from '~/models/dws-billing-statement'
import { DwsBillingStatementAggregate } from '~/models/dws-billing-statement-aggregate'
import { Job } from '~/models/job'
import { api, Api } from '~/services/api/core'

/**
 * 障害福祉サービス：明細書 API.
 */
export namespace DwsBillingStatementsApi {
  type Identifiers = {
    billingId: DwsBillingId
    bundleId: DwsBillingBundleId
  }

  export type GetParams = Api.GetParams & Identifiers
  export type GetResponse = {
    billing: DwsBilling
    bundle: DwsBillingBundle
    statement: DwsBillingStatement
    serviceCodeDictionary: Record<string, string>
  }

  export type UpdateForm = {
    aggregates: Pick<DwsBillingStatementAggregate, 'serviceDivisionCode' | 'managedCopay' | 'subtotalSubsidy'>[]
  }
  export type UpdateParams = Api.UpdateParams<UpdateForm> & Identifiers
  export type UpdateResponse = GetResponse

  export type UpdateCopayCoordinationForm = {
    result: CopayCoordinationResult
    amount: number
  }
  export type UpdateCopayCoordinationParams = Api.UpdateParams<UpdateCopayCoordinationForm> & Identifiers
  export type UpdateCopayCoordinationResponse = GetResponse
  export type UpdateCopayCoordination = {
    updateCopayCoordination (params: UpdateCopayCoordinationParams): Promise<UpdateCopayCoordinationResponse>
  }

  export type UpdateCopayCoordinationStatusForm = {
    status: DwsBillingStatementCopayCoordinationStatus
  }
  export type UpdateCopayCoordinationStatusParams = Api.UpdateParams<UpdateCopayCoordinationStatusForm> & Identifiers
  export type UpdateCopayCoordinationStatusResponse = GetResponse
  export type UpdateCopayCoordinationStatus = {
    updateCopayCoordinationStatus (
      params: UpdateCopayCoordinationStatusParams
    ): Promise<UpdateCopayCoordinationStatusResponse>
  }

  export type UpdateStatusForm = {
    status: DwsBillingStatus
  }
  export type UpdateStatusParams = Api.UpdateParams<UpdateStatusForm> & Identifiers
  export type UpdateStatusResponse = GetResponse
  export type UpdateStatus = {
    updateStatus (params: UpdateStatusParams): Promise<UpdateStatusResponse>
  }
  export type BatchResponse = {
    job: Job
  }
  export type BulkUpdateStatusForm = {
    ids: DwsBillingStatementId[]
    status: DwsBillingStatus
  }
  export type BulkUpdateStatusParams = {
    billingId: DwsBillingId
    form: BulkUpdateStatusForm
  }
  export type BulkUpdateStatus = {
    bulkUpdateStatus (params: BulkUpdateStatusParams): Promise<BatchResponse>
  }
  export type RefreshForm = {
    ids: DwsBillingStatementId[]
  }
  export type RefreshParams = {
    billingId: DwsBillingId
    form: RefreshForm
  }
  export type Refresh = {
    refresh (params: RefreshParams): Promise<BatchResponse>
  }

  export type Definition = Api.Get<GetResponse, GetParams>
    & Api.Update<UpdateForm, UpdateParams, UpdateResponse>
    & UpdateCopayCoordination
    & UpdateCopayCoordinationStatus
    & UpdateStatus
    & BulkUpdateStatus
    & Refresh

  const endpoint = ({ billingId, bundleId, id }: GetParams, last?: string) => {
    return api.endpoint('dws-billings', billingId, 'bundles', bundleId, 'statements', id, last)
  }

  export const create = (axios: AxiosInstance): Definition => ({
    async bulkUpdateStatus ({ billingId, form }) {
      return await api.extract(
        axios.post(api.endpoint('dws-billings', billingId, 'statement-status-update'), form)
      )
    },
    async get (params) {
      return await api.extract(axios.get(endpoint(params)))
    },
    async refresh ({ billingId, form }) {
      return await api.extract(axios.post(api.endpoint('dws-billings', billingId, 'statement-refresh'), form))
    },
    async update (params) {
      return await api.extract(axios.put(endpoint(params), params.form))
    },
    async updateCopayCoordination (params) {
      return await api.extract(axios.put(endpoint(params, 'copay-coordination'), params.form))
    },
    async updateCopayCoordinationStatus (params) {
      return await api.extract(axios.put(endpoint(params, 'copay-coordination-status'), params.form))
    },
    async updateStatus (params) {
      return await api.extract(axios.put(endpoint(params, 'status'), params.form))
    }
  })
}
