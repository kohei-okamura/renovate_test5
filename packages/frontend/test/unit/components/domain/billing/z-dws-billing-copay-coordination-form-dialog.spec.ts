/*
 * Copyright © 2021 EUSTYLE LABORATORY - ALL RIGHTS RESERVED.
 * UNAUTHORIZED COPYING OF THIS FILE, VIA ANY MEDIUM IS STRICTLY PROHIBITED PROPRIETARY AND CONFIDENTIAL.
 */
import { Wrapper } from '@vue/test-utils'
import { CopayCoordinationResult } from '@zinger/enums/lib/copay-coordination-result'
import deepmerge from 'deepmerge'
import Vue from 'vue'
import ZDwsBillingCopayCoordinationFormDialog
  from '~/components/domain/billing/z-dws-billing-copay-coordination-form-dialog.vue'
import { DwsBillingStatementsApi } from '~/services/api/dws-billing-statements-api'
import { ValidationObserverInstance } from '~/support/validation/types'
import { createMockedFormService } from '~~/test/helpers/create-mocked-form-service'
import { getValidationObserver } from '~~/test/helpers/get-validation-observer'
import { setData } from '~~/test/helpers/set-data'
import { setupComponentTest } from '~~/test/helpers/setup-component-test'
import { click } from '~~/test/helpers/trigger'

describe('z-dws-billing-copay-coordination-form-dialog.vue', () => {
  const { mount } = setupComponentTest()
  const $form = createMockedFormService()
  const mocks = {
    $form
  }
  const form: Partial<DwsBillingStatementsApi.UpdateCopayCoordinationForm> = {
    result: CopayCoordinationResult.coordinated,
    amount: 3200
  }
  const propsData = {
    dialog: true,
    amount: 9300,
    errors: {},
    progress: false,
    value: form
  }
  let wrapper: Wrapper<Vue & any>

  function mountComponent (data = propsData) {
    wrapper = mount(ZDwsBillingCopayCoordinationFormDialog, {
      propsData: data,
      mocks
    })
  }

  function unmountComponent () {
    wrapper.destroy()
  }

  describe('initial display', () => {
    it('should be rendered correctly when registration', () => {
      const data = {
        ...propsData,
        value: { result: undefined, amount: undefined }
      }
      mountComponent(data)
      expect(wrapper).toMatchSnapshot()
      unmountComponent()
    })

    it('should be rendered correctly when update', () => {
      mountComponent()
      expect(wrapper).toMatchSnapshot()
      unmountComponent()
    })
  })

  describe('validation', () => {
    type Values = Partial<Omit<typeof form, 'amount'> & { amount?: number | string }>
    let observer: ValidationObserverInstance

    async function validate (cb: () => void, values?: Values) {
      await setData(wrapper, {
        form: deepmerge(form, values ?? {})
      })
      await observer.validate()
      jest.runOnlyPendingTimers()
      cb()
      observer.reset()
    }

    beforeAll(() => {
      mountComponent()
      observer = getValidationObserver(wrapper)
    })

    afterAll(() => {
      unmountComponent()
    })

    it('should pass when input correctly', async () => {
      await validate(
        () => {
          expect(observer).toBePassed()
        }
      )
    })

    it('should fail if result is empty', async () => {
      await validate(
        () => {
          expect(observer).not.toBePassed()
          expect(wrapper.find('[data-result] .v-messages').text()).toBe('入力してください。')
        },
        {
          result: undefined
        }
      )
    })

    it('should fail if amount is empty', async () => {
      await validate(
        () => {
          expect(observer).not.toBePassed()
          expect(wrapper.find('[data-amount] .v-messages').text()).toBe('入力してください。')
        },
        {
          amount: undefined
        }
      )
    })

    it('should fail if amount is not number', async () => {
      await validate(
        () => {
          expect(observer).not.toBePassed()
          expect(wrapper.find('[data-amount] .v-messages').text()).toBe('半角数字のみで入力してください。')
        },
        {
          amount: 'abcde'
        }
      )
    })

    it.each([
      ['appropriated', CopayCoordinationResult.appropriated],
      ['notCoordinated', CopayCoordinationResult.notCoordinated],
      ['coordinated', CopayCoordinationResult.coordinated]
    ])('should not fail if amount is 0 when result is %s', async (_, result) => {
      await validate(
        () => {
          expect(observer).toBePassed()
        },
        {
          result,
          amount: 0
        }
      )
    })
  })

  describe('close', () => {
    beforeAll(() => {
      mountComponent()
    })

    afterAll(() => {
      unmountComponent()
    })

    it('should close dialog when cancel button clicked', async () => {
      await click(() => wrapper.find('[data-cancel]'))
      const events = wrapper.emitted('update:dialog') ?? []
      expect(events).toHaveLength(1)
      expect(events[0]).toStrictEqual([false])
    })
  })
})
