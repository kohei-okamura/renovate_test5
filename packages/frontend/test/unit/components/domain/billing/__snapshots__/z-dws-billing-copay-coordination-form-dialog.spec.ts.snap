// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`z-dws-billing-copay-coordination-form-dialog.vue initial display should be rendered correctly when registration 1`] = `
<v-dialog-stub
  closedelay="0"
  contentclass=""
  opendelay="0"
  origin="center center"
  persistent="true"
  retainfocus="true"
  transition="dialog"
  value="true"
  width="500"
>
  <form
    class="v-form"
    data-form=""
    novalidate="novalidate"
  >
    <v-card-stub
      loaderheight="4"
      tag="div"
    >
      <z-card-titlebar-stub
        color="blue-grey"
      >
        利用者負担上限額管理結果を登録
      </z-card-titlebar-stub>
      <v-card-text-stub
        tag="div"
      >
        <div
          class="subtitle-1 mb-3"
        >
          上限額管理結果を選択してください。
        </div>
        <div>
          <div
            class="root"
            data-result=""
            data-z-form-card-item=""
          >
            <div
              class="v-input theme--light v-input--selection-controls v-input--radio-group v-input--radio-group--column"
            >
              <div
                class="v-input__control"
              >
                <div
                  class="v-input__slot"
                  style="height: auto;"
                >
                  <div
                    class="v-input--radio-group__input"
                    role="radiogroup"
                  >
                    <v-row-stub
                      tag="div"
                    >
                      <v-col-stub
                        tag="div"
                      >
                        <div
                          class="v-radio theme--light"
                        >
                          <div
                            class="v-input--selection-controls__input"
                          >
                            <i
                              aria-hidden="true"
                              class="v-icon notranslate mdi mdi-radiobox-blank theme--light"
                            />
                            <input
                              aria-checked="false"
                              role="radio"
                              type="radio"
                              value="1"
                            />
                            <div
                              class="v-input--selection-controls__ripple"
                            />
                          </div>
                          <label
                            class="v-label theme--light"
                            style="left: 0px; position: relative;"
                          >
                            1. 管理事業所で利用者負担額を充当したため、他事業所の利用者負担は発生しない。
                          </label>
                        </div>
                      </v-col-stub>
                    </v-row-stub>
                    <v-row-stub
                      tag="div"
                    >
                      <v-col-stub
                        tag="div"
                      >
                        <div
                          class="v-radio theme--light"
                        >
                          <div
                            class="v-input--selection-controls__input"
                          >
                            <i
                              aria-hidden="true"
                              class="v-icon notranslate mdi mdi-radiobox-blank theme--light"
                            />
                            <input
                              aria-checked="false"
                              role="radio"
                              type="radio"
                              value="2"
                            />
                            <div
                              class="v-input--selection-controls__ripple"
                            />
                          </div>
                          <label
                            class="v-label theme--light"
                            style="left: 0px; position: relative;"
                          >
                            2. 利用者負担額の合計額が、負担上限月額以下のため、調整事務は行わない。
                          </label>
                        </div>
                      </v-col-stub>
                    </v-row-stub>
                    <v-row-stub
                      tag="div"
                    >
                      <v-col-stub
                        tag="div"
                      >
                        <div
                          class="v-radio theme--light"
                        >
                          <div
                            class="v-input--selection-controls__input"
                          >
                            <i
                              aria-hidden="true"
                              class="v-icon notranslate mdi mdi-radiobox-blank theme--light"
                            />
                            <input
                              aria-checked="false"
                              role="radio"
                              type="radio"
                              value="3"
                            />
                            <div
                              class="v-input--selection-controls__ripple"
                            />
                          </div>
                          <label
                            class="v-label theme--light"
                            style="left: 0px; position: relative;"
                          >
                            3. 利用者負担額の合計額が、負担上限月額を超過するため、下記のとおり調整した。
                          </label>
                        </div>
                      </v-col-stub>
                    </v-row-stub>
                  </div>
                </div>
                <div
                  class="v-messages theme--light"
                >
                  <transition-group-stub
                    class="v-messages__wrapper"
                    name="message-transition"
                    tag="div"
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            class="root pl-8"
            data-amount=""
            data-z-form-card-item=""
          >
            <div
              class="v-input v-input--dense theme--light v-text-field v-text-field--enclosed v-text-field--outlined"
            >
              <div
                class="v-input__control"
              >
                <div
                  class="v-input__slot"
                  style="background-color: rgb(241, 246, 250); border-color: #f1f6fa;"
                >
                  <fieldset
                    aria-hidden="true"
                  >
                    <legend
                      style="width: 0px;"
                    >
                      <span
                        class="notranslate"
                      >
                        ​
                      </span>
                    </legend>
                  </fieldset>
                  <div
                    class="v-text-field__slot"
                  >
                    <label
                      class="v-label theme--light"
                      style="left: 0px; position: absolute;"
                    >
                      上限管理後利用者負担額 *
                    </label>
                    <input
                      min="0"
                      type="number"
                    />
                    <div
                      class="v-text-field__suffix"
                    >
                      円
                    </div>
                  </div>
                </div>
                <div
                  class="v-text-field__details"
                >
                  <div
                    class="v-messages theme--light"
                  >
                    <transition-group-stub
                      class="v-messages__wrapper"
                      name="message-transition"
                      tag="div"
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </v-card-text-stub>
      <v-card-actions-stub
        class="pb-4 pt-0 px-4"
        tag="div"
      >
        <v-spacer-stub
          tag="div"
        />
        <a
          class="v-btn-stub"
          data-cancel=""
          text=""
        >
          キャンセル
        </a>
        <a
          class="v-btn-stub"
          color="primary"
          data-ok=""
          depressed=""
          type="submit"
        >
          登録
        </a>
      </v-card-actions-stub>
    </v-card-stub>
  </form>
</v-dialog-stub>
`;

exports[`z-dws-billing-copay-coordination-form-dialog.vue initial display should be rendered correctly when update 1`] = `
<v-dialog-stub
  closedelay="0"
  contentclass=""
  opendelay="0"
  origin="center center"
  persistent="true"
  retainfocus="true"
  transition="dialog"
  value="true"
  width="500"
>
  <form
    class="v-form"
    data-form=""
    novalidate="novalidate"
  >
    <v-card-stub
      loaderheight="4"
      tag="div"
    >
      <z-card-titlebar-stub
        color="blue-grey"
      >
        利用者負担上限額管理結果を編集
      </z-card-titlebar-stub>
      <v-card-text-stub
        tag="div"
      >
        <div
          class="subtitle-1 mb-3"
        >
          上限額管理結果を選択してください。
        </div>
        <div>
          <div
            class="root"
            data-result=""
            data-z-form-card-item=""
          >
            <div
              class="v-input v-input--is-label-active v-input--is-dirty theme--light v-input--selection-controls v-input--radio-group v-input--radio-group--column"
            >
              <div
                class="v-input__control"
              >
                <div
                  class="v-input__slot"
                  style="height: auto;"
                >
                  <div
                    class="v-input--radio-group__input"
                    role="radiogroup"
                  >
                    <v-row-stub
                      tag="div"
                    >
                      <v-col-stub
                        tag="div"
                      >
                        <div
                          class="v-radio theme--light"
                        >
                          <div
                            class="v-input--selection-controls__input"
                          >
                            <i
                              aria-hidden="true"
                              class="v-icon notranslate mdi mdi-radiobox-blank theme--light"
                            />
                            <input
                              aria-checked="false"
                              role="radio"
                              type="radio"
                              value="1"
                            />
                            <div
                              class="v-input--selection-controls__ripple"
                            />
                          </div>
                          <label
                            class="v-label theme--light"
                            style="left: 0px; position: relative;"
                          >
                            1. 管理事業所で利用者負担額を充当したため、他事業所の利用者負担は発生しない。
                          </label>
                        </div>
                      </v-col-stub>
                    </v-row-stub>
                    <v-row-stub
                      tag="div"
                    >
                      <v-col-stub
                        tag="div"
                      >
                        <div
                          class="v-radio theme--light"
                        >
                          <div
                            class="v-input--selection-controls__input"
                          >
                            <i
                              aria-hidden="true"
                              class="v-icon notranslate mdi mdi-radiobox-blank theme--light"
                            />
                            <input
                              aria-checked="false"
                              role="radio"
                              type="radio"
                              value="2"
                            />
                            <div
                              class="v-input--selection-controls__ripple"
                            />
                          </div>
                          <label
                            class="v-label theme--light"
                            style="left: 0px; position: relative;"
                          >
                            2. 利用者負担額の合計額が、負担上限月額以下のため、調整事務は行わない。
                          </label>
                        </div>
                      </v-col-stub>
                    </v-row-stub>
                    <v-row-stub
                      tag="div"
                    >
                      <v-col-stub
                        tag="div"
                      >
                        <div
                          class="v-radio theme--light v-item--active"
                        >
                          <div
                            class="v-input--selection-controls__input"
                          >
                            <i
                              aria-hidden="true"
                              class="v-icon notranslate mdi mdi-radiobox-marked theme--light primary--text"
                            />
                            <input
                              aria-checked="true"
                              role="radio"
                              type="radio"
                              value="3"
                            />
                            <div
                              class="v-input--selection-controls__ripple primary--text"
                            />
                          </div>
                          <label
                            class="v-label theme--light"
                            style="left: 0px; position: relative;"
                          >
                            3. 利用者負担額の合計額が、負担上限月額を超過するため、下記のとおり調整した。
                          </label>
                        </div>
                      </v-col-stub>
                    </v-row-stub>
                  </div>
                </div>
                <div
                  class="v-messages theme--light"
                >
                  <transition-group-stub
                    class="v-messages__wrapper"
                    name="message-transition"
                    tag="div"
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            class="root pl-8"
            data-amount=""
            data-z-form-card-item=""
          >
            <div
              class="v-input v-input--is-label-active v-input--is-dirty v-input--dense theme--light v-text-field v-text-field--enclosed v-text-field--outlined"
            >
              <div
                class="v-input__control"
              >
                <div
                  class="v-input__slot"
                  style="background-color: rgb(241, 246, 250); border-color: #f1f6fa;"
                >
                  <fieldset
                    aria-hidden="true"
                  >
                    <legend
                      style="width: 0px;"
                    >
                      <span
                        class="notranslate"
                      >
                        ​
                      </span>
                    </legend>
                  </fieldset>
                  <div
                    class="v-text-field__slot"
                  >
                    <label
                      class="v-label v-label--active theme--light"
                      style="left: 0px; position: absolute;"
                    >
                      上限管理後利用者負担額 *
                    </label>
                    <input
                      min="0"
                      type="number"
                    />
                    <div
                      class="v-text-field__suffix"
                    >
                      円
                    </div>
                  </div>
                </div>
                <div
                  class="v-text-field__details"
                >
                  <div
                    class="v-messages theme--light"
                  >
                    <transition-group-stub
                      class="v-messages__wrapper"
                      name="message-transition"
                      tag="div"
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </v-card-text-stub>
      <v-card-actions-stub
        class="pb-4 pt-0 px-4"
        tag="div"
      >
        <v-spacer-stub
          tag="div"
        />
        <a
          class="v-btn-stub"
          data-cancel=""
          text=""
        >
          キャンセル
        </a>
        <a
          class="v-btn-stub"
          color="primary"
          data-ok=""
          depressed=""
          type="submit"
        >
          編集
        </a>
      </v-card-actions-stub>
    </v-card-stub>
  </form>
</v-dialog-stub>
`;
